wp-cldr.php

```diff
@@ -33,20 +33,28 @@ function wp_cldr_menu() {
 }
 
 /**
+ * Loads the text domain for the plug-in. We call this only when the settings page is loaded since translations aren't needed any earlier.
+ */
+function wp_cldr_load_plugin_textdomain() {
+	load_plugin_textdomain( 'wp-cldr', false, basename( dirname( __FILE__ ) ) . '/languages/' );
+}
+
+/**
  * Gets the settings and examples page for the plugin.
  */
 function wp_cldr_settings() {
 
+	wp_cldr_load_plugin_textdomain();
```

Comment:
Do you anticipate calling this anywhere else? If not, I'd say just call `load_plugin_textdomain` here instead.